File: Sis/Unidad8/Ejercicio1/Ejercicio1/Hubs/clsMensajeUsuario.cs
namespace SignalRChat.Hubs
{
    public class clsMensajeUsuario
    {

        public string usuario {  get; set; }
        public string mensaje { get; set; }

        private clsMensajeUsuario()
        {
            this.usuario = "";
            this.mensaje = "";
        }
        private clsMensajeUsuario(string usuario, string mensaje)
        {
            this.usuario=usuario;
            this.mensaje=mensaje;
        }
      
    }
}

File: Sis/Unidad8/Ejercicio1/Ejercicio1/wwwroot/js/clsMensajeUsuario.js
namespace SignalRChat.wwwroot.js
{
   // wwwroot/js/clsMensajeUsuario.js
export class clsMensajeUsuario {
    constructor(nombre, mensaje) {
        this.nombre = nombre;
        this.mensaje = mensaje;
    }

    getNombre() {
        return this.nombre;
    }

    setNombre(nombre) {
        this.nombre = nombre;
    }

    getMensaje() {
        return this.mensaje;
    }

    setMensaje(mensaje) {
        this.mensaje = mensaje;
    }
}
}
